frontend/src/features/update-workflow/ui/nodes/editableNodes.module.css

```
@@ -1,217 +1,329 @@
+.description {
+  color: var(--text-secondary);
+  /* Match the viewer's .description — Pretendard base (340) is too thin for
+     the small node card; 450 stays comfortably below heading weight (540)
+     while improving readability. */
+  font-weight: 450;
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
+  line-clamp: 2;
+  -webkit-box-orient: vertical;
+  overflow: hidden;
+  word-break: break-word;
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)